README.md

```diff
@@ -122,3 +143,4 @@ Places to ask questions and find other community resources.
 - [Bayerische Ehrenamtskarte](https://github.com/digitalfabrik/ehrenamtskarte): Fully open-source solution for entitlement cards
 - [AusweisApp2](https://www.ausweisapp.bund.de/ausweisapp2/): app for the german identity card (*neuer Personalausweis*, nPA)
   - [GitHub](https://github.com/Governikus/AusweisApp2)
+- [bflow](http://bflow.org/): OSS based business process modelling
```

Comment:
How is this relevant for public administration (esp. in comparison with other process modeling tools like, e.g., https://github.com/camunda)?